azerothcore/azerothcore-wotlk
# DB/Misc: Update some VoA scripts SAI, add waypoints

## CHANGES PROPOSED:
-  Updated the SAI of creatures and the timers "sniffs"
- Add waypoints from sniffs
- Fix amount of gold from Flame Warder in heroic

## TESTS PERFORMED:
- Build and test in game


## HOW TO TEST THE CHANGES:

- .tele VaO
- Go to instant 
- See move wayponts Mobs and go fight him and look cast spell. 

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
